components/bread-crumb/index.jsx

```diff
@@ -3,5 +3,11 @@
 
 // Alias
 import Breadcrumb from '../breadcrumb';
+import componentHasMoved from '../../utilities/warning/component-has-moved';
+
+componentHasMoved('SLDSBreadCrumb', {
```

Comment:
These should use the component name from `./utilities/constants.js`.

Reply:
Updated.